datasource/threedi_results.py

```diff
@@ -99,47 +44,20 @@ def __init__(self, file_path=None):
         self._datasource = None
         self._cache = {}
 
-    @property
-    def ds(self):
-        # TODO: move to constructor or make cached_property
-        if self._datasource is None:
-            try:
-                self._datasource = h5py.File(self.file_path, "r")
-            except IOError as e:
-                logger.exception(e)
-                raise e
-        return self._datasource
-
-    @property
-    def nMesh2D_nodes(self):
-        return self.gridadmin.nodes.subset("2D_ALL").count
-
-    @property
-    def nMesh1D_nodes(self):
-        return self.gridadmin.nodes.subset("1D_ALL").count
-
-    @property
-    def nMesh2D_lines(self):
-        return self.gridadmin.lines.subset("2D_ALL").count
-
-    @property
-    def nMesh1D_lines(self):
-        return self.gridadmin.lines.subset("1D_ALL").count
-
     @cached_property
     def available_subgrid_map_vars(self):
         """Return a list of available variables from 'results_3di.nc'."""
         known_subgrid_map_vars = set([v.name for v in SUBGRID_MAP_VARIABLES])
         if self.gridadmin_result.has_pumpstations:
             available_vars = (
-                self.gridadmin_result.nodes._field_names
-                | self.gridadmin_result.lines._field_names
-                | self.gridadmin_result.pumps._field_names
+                    self.gridadmin_result.nodes._field_names
```

Comment:
Dit zal niet aan deze PR letten, maar..... underscore variabelen zijn normaliter expliciet verboden terrein voor externen.
En nu zitten we in underscore variabelen van een variabele van een variabele te "wroeten"...

Is dit het goede gebruik van de gridadmin?